packages/@ember/array/index.ts

```diff
@@ -792,6 +790,10 @@ interface EmberArray<T> extends Enumerable {
     @return {Object} Found item or `undefined`.
     @public
   */
+  find<S extends T, Target = void>(
```

Comment:
Note: conventionally/idiomatically it's `U` in a case like this (since T → U in the alphabet). Not a thing I care about changing, to be sure, just worth noting!

Reply:
I think I just copied this from somewhere else :)